test/index.js

```diff
@@ -648,6 +648,40 @@ describe('def', function() {
     eq(sinceEpoch(new Date(123456)), 123.456);
   });
 
+  it('supports the "PositiveNumber" type', function() {
+    eq($.PositiveNumber.name, 'sanctuary-def/PositiveNumber');
+    eq($.PositiveNumber.test(null), false);
+    eq($.PositiveNumber.test(NaN), false);
+    eq($.PositiveNumber.test(new Number(NaN)), false);
+    eq($.PositiveNumber.test(-1), false);
+    eq($.PositiveNumber.test(new Number(-1)), false);
+    eq($.PositiveNumber.test(0), false);
+    eq($.PositiveNumber.test(new Number(0)), false);
+    eq($.PositiveNumber.test(-0), false);
+    eq($.PositiveNumber.test(new Number(-0)), false);
+    eq($.PositiveNumber.test(0.5), true);
+    eq($.PositiveNumber.test(new Number(0.5)), true);
+    eq($.PositiveNumber.test(Infinity), true);
+    eq($.PositiveNumber.test(new Number(Infinity)), true);
+  });
+
+  it('supports the "NegativeNumber" type', function() {
+    eq($.NegativeNumber.name, 'sanctuary-def/NegativeNumber');
+    eq($.NegativeNumber.test(null), false);
+    eq($.NegativeNumber.test(NaN), false);
+    eq($.NegativeNumber.test(new Number(NaN)), false);
+    eq($.NegativeNumber.test(1), false);
+    eq($.NegativeNumber.test(new Number(1)), false);
+    eq($.NegativeNumber.test(0), false);
```

Comment:
I recommend keeping the pull requests separate, in case the convenience function is contentious. ;)